agent/06_agent_create_cluster.sh

```diff
@@ -152,6 +162,45 @@ function attach_appliance_diskimage() {
     done
 }
 
+function attach_agent_iso_no_registry() {
+    set_file_acl
+
+    local base_dir=$SCRIPTDIR/$OCP_DIR
+    local iso_name="agent-ove.$(uname -p).iso"
+    local agent_iso_no_registry=$(find "$base_dir" -type f -name "$iso_name" 2>/dev/null | head -n 1)
+
+    for (( n=0; n<${2}; n++ ))
+    do
+        name=${CLUSTER_NAME}_${1}_${n}
+        sudo virt-xml ${name} --add-device --disk "${agent_iso_no_registry}",device=cdrom,target.dev=sdc
+        sudo virt-xml ${name} --edit target=sda --disk="boot_order=1"
+        sudo virt-xml ${name} --edit target=sdc --disk="boot_order=2" --start
+    done
+}
+
+function automate_rendezvousIP_selection(){
+  for (( n=0; n<${2}; n++ ))
+    do
+        name=${CLUSTER_NAME}_${1}_${n}
+        ./agent/e2e/agent-tui/automate-no-registry-agent-tui.sh $name
```

Comment:
We should take console screenshots before and after the script.
Example:
 [     sudo virsh screenshot $name "${OCP_DIR}/${name}_console_screenshot_before_dns_fix.ppm" ](https://github.com/openshift-metal3/dev-scripts/blob/39a6bd2cad1d811b090c1e78a08b35d8cd03962e/agent/06_agent_create_cluster.sh#L296C4-L296C91)